.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -163,8 +163,8 @@
         </ProcessMauiFontsAfterTargets>
     </PropertyGroup>
 
-    <!-- UWP / WinUI -->
-    <PropertyGroup Condition="'$(_ResizetizerIsUWPApp)' == 'True' Or '$(_ResizetizerIsWindowsAppSdk)' == 'True'">
+    <!-- Windows App SDK -->
+    <PropertyGroup Condition="'$(_ResizetizerIsWindowsAppSdk)' == 'True'">
```

Comment:
Just cleaning up comments and conditions. The `$(_ResizetizerIsUWPApp)` property was removed some time ago.